entity/fitsQAUberCreateEntity.java

```diff
@@ -0,0 +1,126 @@
+
+        Thread.sleep(5000);
+// Log-In
+        WebElement LogBn = dr.findElement(By.xpath("/html/body/div[2]/div/div/div[1]/div[2]/form/button"));
+        LogBn.click();
+
+//Navigate Entities
+        WebElement Entities = dr.findElement(By.xpath("/html/body/div[2]/aside[1]/div/nav/ul/li[2]/a/span"));
+        Entities.click();
+
+        for (int i = 0; i < 10; i++)
+        {
+//Add Entity
+            WebElement AddEntity = dr.findElement(By.xpath("/html/body/div[2]/section/div/h3/button"));
+            AddEntity.click();
+//Create Entity
+            WebElement Entityname = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[1]/div[1]/form/div[1]/div[1]/div/input"));
+            String Ename = faker.name().firstName();
+            Entityname.sendKeys(Ename);
+            WebElement Phone = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[1]/div[1]/form/div[1]/div[2]/div/input"));
+            Phone.sendKeys("44444444");
+            WebElement EMail = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[1]/div[1]/form/div[1]/div[3]/div/input"));
+            EMail.sendKeys("xxxsssx@gmail.com");
+            //WebElement EntityLogo = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[1]/div[1]/form/div[2]/div[1]/div[1]/div/div[2]/button[1]/span[1]"));
+            WebElement Country = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[1]/div[1]/form/div[2]/div[1]/div[2]/select"));
+            Country.click();
+            dr.manage().timeouts().implicitlyWait(100, TimeUnit.SECONDS);
+            Country.submit();
+            Country.sendKeys("United States");
+            WebElement Address1 = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[1]/div[1]/form/div[2]/div[1]/div[3]/input"));
+            Address1.sendKeys("XXXXXX");
+            WebElement City = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[1]/div[1]/form/div[2]/div[1]/div[5]/input"));
+            City.sendKeys("XXXXXX");
+            Country.sendKeys("United States");
+            WebElement State = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[1]/div[1]/form/div[2]/div[1]/div[6]/select"));
+            State.sendKeys("California");
+            WebElement PostalCode = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[1]/div[1]/form/div[2]/div[1]/div[7]/input"));
+            PostalCode.sendKeys("43524");
+            WebElement NextStep = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[2]/button[2]/span[1]"));
+            NextStep.click();
+
+            WebElement NextStep2 = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[2]/button[2]"));
+            NextStep2.click();
+
+            WebElement NextStep3 = dr.findElement(By.xpath("/html/body/div[2]/section/div/div/div[2]/button[2]"));
+            NextStep3.click();
+//Slider
+            //    /html/body/div[2]/section/div/div/div[1]/div[4]/div/div/div[1]/div[2]
```

Comment:
remove comments. cleanup the code

Reply:
Unwanted comments removed as per clean up.